bugsnag/src/main/java/com/bugsnag/serialization/ISerializer.java

```diff
@@ -0,0 +1,23 @@
+package com.bugsnag.serialization;
+
+import java.io.OutputStream;
+
+public interface ISerializer {
```

Comment:
We should be fine to call this `Serializer` and rename the existing implementation as `DefaultSerializer` - although it was a public class, there was no way to use it so we can replace the class name with the interface without issue.